### PR TITLE
add array support to several blacklist ip commands

### PR DESCRIPTION
**Pull request by esharf**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/41981

## Description
Added array support for the following commands:
- in F5 Silverline integration:
    - f5-silverline-ip-object-add
- in Signal Sciences WAF integration:
    - sigsci-blacklist-add-ip
    - sigsci-whitelist-add-ip
 - in ThreatX integration:
     - threatx-blacklist-ip

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Tests

### CODE REVIEW

File: Packs/F5Silverline/ReleaseNotes/1_0_6.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### F5 Silverline
+- Added array support for **f5-silverline-ip-object-add** command.
```
Comment (yaakovi) [inline Packs/F5Silverline/ReleaseNotes/1_0_6.md:4]: update all RN by this template.
```suggestion
- The *cidr_range* argument for the **f5-silverline-ip-object-add** command now supports a list of ranges.
```

File: Packs/SignalSciences/Integrations/SignalSciences/SignalSciences.py [hunk 2]
```diff
@@ -418,7 +418,7 @@ def adjust_alert_human_readable(entry_context_with_spaces, entry_context):
 
 def check_ip_is_valid(ip):
     if not is_ip_valid(ip):
-        return_error("Error: IP argument is invalid. Please enter a valid IP address")
+        return_error("Error: {} is invalid IP. Please enter a valid IP address")
```
Comment (yaakovi) [inline Packs/SignalSciences/Integrations/SignalSciences/SignalSciences.py:421]:
```suggestion
        return_error(f"Error: {ip} is invalid IP. Please enter a valid IP address")
```

File: Packs/SignalSciences/Integrations/SignalSciences/SignalSciences.py [hunk 3]
```diff
@@ -445,42 +445,50 @@ def gen_entries_data_for_update_list_request(entries_list, method):
...
+            data = {
+                'source': single_ip,
+                'note': note
+            }
+            if expires is not None:
+                data['expires'] = expires
+            res_list.append(http_request('PUT', url, data=data))
+        except SystemExit:
+            # handle exceptions in return_error
+            pass
```
Comment (yaakovi) [inline Packs/SignalSciences/Integrations/SignalSciences/SignalSciences.py:487]: no need to check that, the first failure should stop the command (no need to continue to the rest of the least).

relevant for all.

**Review by yaakovi — COMMENTED** [review]
* Please add unit-tests in the additional integrations
* update the description of the relevant arguments in the yml and in the README files.

File: Packs/SignalSciences/Integrations/SignalSciences/SignalSciences.py (hunk 3 above)
Comment (esharf) [reply Packs/SignalSciences/Integrations/SignalSciences/SignalSciences.py:487]: i have fixed it in the other integrations.
but in this one specific it’s hard to fix it.

**Comment by esharf** [comment]
@yaakovi the PR is ready to merge (will require a force merge)

File: Packs/ThreatX/ReleaseNotes/1_0_6.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### ThreatX
+- Added array support for **threatx-blacklist-ip** command.
```
Comment (yaakovi) [inline Packs/ThreatX/ReleaseNotes/1_0_6.md:4]:
```suggestion
- Added array support for the ***threatx-blacklist-ip*** command.
```

File: Packs/SignalSciences/ReleaseNotes/1_0_7.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Signal Sciences WAF
+- Added array support for **sigsci-blacklist-add-ip** command.
+- Added array support for **sigsci-whitelist-add-ip** command.
```
Comment (yaakovi) [inline Packs/SignalSciences/ReleaseNotes/1_0_7.md:5]:
```suggestion
- Added array support for the ***sigsci-blacklist-add-ip*** command.
- Added array support for the ***sigsci-whitelist-add-ip*** command.
```

File: Packs/F5Silverline/ReleaseNotes/1_0_6.md [hunk 6]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### F5 Silverline
+- The *cidr_range* argument for the **f5-silverline-ip-object-add** command now supports a list of ranges.
```
Comment (yaakovi) [inline Packs/F5Silverline/ReleaseNotes/1_0_6.md:4]:
```suggestion
- The *cidr_range* argument for the ***f5-silverline-ip-object-add*** command now supports a list of ranges.
```

**Review by yaakovi — APPROVED** [review]

**Pull request merged**